javademo/src/main/java/com/example/javademo/Main.java

```diff
@@ -31,14 +35,21 @@ public static void main(String[] args) throws Exception {
         Path contractPath = Paths.get("balance.cairo");
         String contract = String.join("", Files.readAllLines(contractPath));
 
-        // Deploy a contract
+        // Declare a contract
+        // Class hash has to be calculated manually
         ContractDefinition contractDefinition = new ContractDefinition(contract);
-        DeployTransactionPayload payload = new DeployTransactionPayload(contractDefinition, Felt.fromHex("0x1234"), Collections.emptyList(), Felt.ZERO);
-        Request<DeployResponse> deployRequest = provider.deployContract(payload);
-        DeployResponse deployResponse = deployRequest.send();
+        Felt classHash = Felt.fromHex("0x399998c787e0a063c3ac1d2abac084dcbe09954e3b156d53a8c43a02aa27d35");
```

Comment:
I would use some address clearly indicating that it's a fake one. Like `0x1234`.